tests/test_staticfiles.py

```diff
@@ -441,3 +441,28 @@ def mock_timeout(*args, **kwargs):
     response = client.get("/example.txt")
     assert response.status_code == 500
     assert response.text == "Internal Server Error"
+
+
+def test_staticfiles_follows_symlinks_to_break_out_of_dir(
```

Comment:
I've run this test without your changes, and this test still passes. I guess this is not expected...

Reply:
Don't we want the symlink file to be inside the directory, and the real file to be outside?

Reply:
Yes you're right, the test should be the other way around.